### ISSUE TITLE
Show more info about a users reputation score in profile section

### ISSUE THREAD

**Issue by darawhelan** [bisq-network/bisq2#1476]
Bisq2 should include more information for sellers for what they need to do to increase their ranking.

![Reputation2](https://github.com/bisq-network/bisq2/assets/121062323/97f754ad-97c6-4a48-be7c-973aa52d14c2)

Under reputation score include:

- Star rating
text description `You are currently rated 2 stars, Sellers with higher reputation get more trades. To get to 3 stars you need to Burn XXX BSQ or Bond XXXX BSQ `

- Reputation ranking
text description 

`You currently have no reputation`

`You are currently ranked #23 out of 59 sellers. To get into the top 10 seller you would need to Burn XXX BSQ or Bond XXXX BSQ` 

`Congratulations you are a top 10 ranked seller!!! You are ranked #9 out of 59 sellers. To get to number 1 you would need to Burn XXX BSQ or Bond XXXX BSQ`  

`Congratulations you are are Bisq top ranked seller!!! You are ranked #1 out of 59 sellers. To keep your number 1 remember to regularly Burn XXX BSQ or Bond XXXX BSQ`

**Comment by mastakalandir** [comment]
Hello! Where can I find the design files to duplicate and begin working on things?

**Comment by darawhelan** [comment]
@HenrikJannsen please can you let @mastakalandir where he can access the design files

**Comment by HenrikJannsen** [comment]
There is no design for that. myabe @axpoems have some suggestions.

**Comment by axpoems** [comment]
@darawhelan thanks for your inputs.

I had in mind working on a revamp of the user profile screen in general. I can include your feedback and upload the proposal mockup here, which can include as you say, more focus on the reputation of the user.

However we are now focused on the release of Bisq2 and working towards milestone 2.0.0. so I don't have immediate availability for this. Nonetheless, I'll try to work on this soon.

In which way would you like to contribute @mastakalandir: in the UI/UX design, development..?

**Comment by axpoems** [comment]
I propose the following changes:

* Remove the reputation from the "User Profile" tab and add a dedicated "My Reputation" tab specifically dedicated to the reputation details.
* "My reputation" then seeks to make the user understand how the score is calculated, all the factors taken into account and can easily click on the buttons to increase the items that available for it.
* Added a "new" label in order to make more visible the new additions.
* Make the stars always available next to the profile icon (main icon in the tab views and top right).
* The sentences ofc only appear if it applies.

![image](https://github.com/bisq-network/bisq2/assets/145597137/6f1f8d7d-c8a6-4926-beb2-713f5a01e2e4)

![image](https://github.com/bisq-network/bisq2/assets/145597137/4e986a21-5cc4-4e05-84c4-cc75b9daa6d0)

I will work on the star system and add the mock-ups around that in a different issue.

@HenrikJannsen @darawhelan  - Let me know your feedback.

**Comment by HenrikJannsen** [comment]
Very good idea!

**Comment by HenrikJannsen** [comment]
Profile age is not counting for the score but is only shown as informational addition as it is somehow reputation related. Not sure if we should remove it or leave it. We have it in the other rep. details view as well....

**Comment by axpoems** [comment]
> Profile age is not counting for the score but is only shown as informational addition as it is somehow reputation related. Not sure if we should remove it or leave it. We have it in the other rep. details view as well....
> Maybe in the buttons column a info icon with tooltip or info popup might be useful?

Good point. Then I will just add this info in the user profile details tab, and remove it from here.

**Comment by axpoems** [comment]
I will add these changes.

**Comment by darawhelan** [comment]
@axpoems 

That looks great.

Where will the Reputation Ranking table be moved to? perhaps this should have its own tab after 'My Reputation'?

**Comment by axpoems** [comment]
> Where will the Reputation Ranking table be moved to? perhaps this should have its own tab after 'My Reputation'?

See my proposal for this here: https://github.com/bisq-network/bisq2/issues/1835#issuecomment-2016947797
